### DISCUSSION TITLE
SMTP Configuration By Cloud Plan?

### DISCUSSION THREAD

**Discussion by pedrolpz99** [docusealco/docuseal#347] [General]
I'm trying to set the SMTP configuration by my email provider, currently my plan is "Cloud" and all the emails come from docuseal, but i want to change it by my domain, my email provider is mailgun, so there is a way to configure this option or is only for On-premises plan?

**Comment by omohokcoj** [comment]
Hi @pedrolpz99 at the moment SMTP configurations are available only with On-premises hosted app.
In the Cloud we provide Gmail/Outlook integration to send emails from the business email address:
https://www.docuseal.co/resources/connect-gmail-or-outlook

**Reply by pedrolpz99** [reply]
there is a reason for why it's only for On-premises hosted app?, what can i do if i don't have google workspace or Outlook?

**Reply by omohokcoj** [reply]
@pedrolpz99 unfortunately Outlook or Gmail integration is the only option to send company emails in the Cloud.
https://www.microsoft.com/en-us/microsoft-365/business/compare-all-microsoft-365-business-products - Outlook business email requires Business Basic subscription in order to connect the company domain
